edml-1.0.0.json

```diff
@@ -0,0 +1,215 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "https://schemas.exasol.com/edml-1.0.0.json",
+  "definitions": {
+    "varcharColumnSize": {
+      "type": "integer",
+      "default": 254,
+      "minimum": 1
+    },
+    "fields": {
+      "type": "object",
+      "title": "",
+      "description": "Using the fields object definition a step down into the property tree is modeled. \n\n**Properties**:\n\nNAME_OF_DYNAMODB_COLUMN: mappingDefinition\n\nIt must be specified a least one column.",
+      "minProperties": 1,
+      "additionalProperties": {
+        "$ref": "#/definitions/mappingDefinition"
+      }
+    },
+    "destinationTable": {
+      "type": "string",
+      "title": "Name of the destination table in Exasol Virtual Schema",
+      "description": "The name of the resulting table in Exasol.",
+      "pattern": "^[a-zA-Z0-9_]+$"
```

Comment:
Exasol allows a lot of UTF-8 characters in object IDs:
https://docs.exasol.com/sql_references/basiclanguageelements.htm#SQLIdentifier

Reply:
```suggestion
```
So I would sugest just to remove the regex. In that case Exasol will throw an error if the name does not fit.

Reply:
You could also "borrow" the regex-parts from this code here:
https://github.com/exasol/db-fundamentals-java/blob/master/src/main/java/com/exasol/db/ExasolIdentifier.java
It's going to be a long regex, but still a nice feature.